docker/utils.sh

```diff
@@ -64,35 +65,33 @@ function parse_git_branch {
 }
 
 PS1='${USER_STR} :: ${WORKDIR_STR} ${GIT_STR}$ '
-" >> "${HOME_DIR}/.bashrc"
+" >> "$HOME/.bashrc"
 }
 
-config_jupyter() {
+configure_jupyter() {
 # Configure Jupyter
 # Set --no-browser --ip=0.0.0.0
-  local user_name="$1"
-  local home_dir="$2"
-
-  if [ "$user_name" = "root" ]
-  then
-    jupyter-lab --generate-config
-  else
-    su "$user_name" -c 'jupyter-lab --generate-config'
-  fi 
-  sed -i"" \
```

Comment:
Just for my understanding, why do you remove the option for the user `root` ? Same for `download_nltk` function ?

Reply:
Yes, that's because it was changed how the functions are called. Before they were called by root, so you had to use `su` inside the function, but now they are called directly by the appropriate user, so you don't need `su`.

Just to give a toy example:

Before:
```shell
function do_something() {
  local user_name="$1"
  if [ "$user_name" = "root" ]
  then
    echo "I'm root"
  else
    su "$user_name" -c "echo I'm $user_name"
  fi 
}

# Run do_something as root, then change to $user_name inside of do_something
user_name="<whatever>"
do_something $user_name
```

After
```shell
function do_something() {
  echo "I'm $USER"
}

# Run do_something as user $user_name straight away
user_name="<whatever>"
su $user_name -c "$(declare -f do_something) && do_something"
```

Reply:
I see! Thanks for the explanation @Stannislav